Security/src/ExchangeExtendedProtectionManagement/DataCollection/Get-ExtendedProtectionPrerequisitesCheck.ps1

```diff
@@ -60,6 +65,18 @@ function Get-ExtendedProtectionPrerequisitesCheck {
                 Write-Progress @progressParams
                 Write-Verbose "$($progressParams.Status)"
                 $tlsSettings = Get-AllTlsSettings -MachineName $server.FQDN
+                $params = @{
+                    MachineName = $server.FQDN
+                    SubKey      = "SYSTEM\CurrentControlSet\Control\Lsa"
+                }
+
+                $lmValue = Get-RemoteRegistryValue @params -GetValue "LmCompatibilityLevel" -ValueType "DWord"
+                [int]$epValue = Get-RemoteRegistryValue @params -GetValue "SuppressExtendedProtection"
+
+                if ($null -eq $lmValue) { $lmValue = 3 }
+
+                $registryValues.SuppressExtendedProtection = $epValue
+                $registryValues.LmCompatibilityLevel = $lmValue
```

Comment:
@dpaulson45 we query the `LmCompatibilityLevel` value but don't write the result out or use it for further testing. Is this something that we'll do with an future update?